python/methods.md

```diff
@@ -106,7 +107,9 @@ Methods can be called in several ways.
 >>> add(1, y=2)
 >>> add(y=2, x=1)
 >>> add(y=2, 1)
->>> add(y=2, 1)
+  File "<stdin>", line 1
+SyntaxError: non-keyword arg after keyword arg
+>>> add(y=2, =1)
```

Comment:
There's a syntax error here, `, =1`.

Reply:
That was already in the course, on purpose I guess. But there were some mistakes too, including a stack trace that did not correspond. I fixed both things - no problem in showing a mistake here and there, I reckon.

Reply:
To be clear: now the examples are either correct or bad usecases and the latter show a corresponding stack trace.